website/docs/workflows/reference/workflow-triggers.md

```diff
@@ -0,0 +1,82 @@
+---
+description: This page provides detailed information on workflow triggers available in Appsmith.
+title: Workflow Triggers  
+hide_title: true  
+toc_min_heading_level: 2  
+toc_max_heading_level: 3  
+---
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Workflow Triggers</h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+
+Workflow triggers start workflow runs based on specific conditions or events. This page provides information about the workflow triggers available in Appsmith, including their configurations, parameters, and usage examples.
+
+## Webhook trigger
+
+A webhook trigger allows external services to start a workflow by sending an HTTP request to a specific URL. This trigger is useful for integrating with applications and services that can send HTTP requests when certain events occur. For example, you can set up a webhook trigger to automatically start a workflow whenever a user signs up on your website, allowing you to send a welcome email or update a database in real time.
+
+### Enable webhook trigger
+
+Toggle the switch to enable the webhook trigger. Once enabled, Appsmith generates a unique identifier URL for the workflow.
+
+### URL
+
+Copy the provided endpoint URL to configure the webhook in your external service. This unique URL includes an API key for authentication and is not shown again for security reasons, so be sure to save it for future reference. If you forget to note the URL, you can regenerate it and must update all applications or services to use the new URL. To configure the workflow, use the HTTP POST method along with the webhook URL. For more information, see [Create Basic Workflow](/workflows/tutorials/create-workflow) using the webhook trigger.
+
+## Scheduled trigger
+
+A scheduled trigger starts a workflow at specific times or regular intervals, making it useful for automating tasks that need to run on a fixed schedule. For example, you can set up a scheduled trigger to automatically send a daily report summarizing sales performance to the sales team. 
+
+### Timezone
+
+Select the appropriate timezone from the dropdown menu. The default value is the system's timezone, such as `UTC+05:30 - Asia/Kolkata`. Ensure that the selected timezone aligns with when the workflow runs.
+
+### Set schedule
+
+The workflow schedule uses a standard 5-field cron expression to define when the workflow runs. To schedule a workflow, configure the fields—**Minute**, **Hour**, **Day of Month**, **Month**, and **Day of Week**.
+
+You can use the following syntax in each field to configure the schedule:
+
+- **Asterisk (`*`)**: Allows you to include a wildcard that represents all possible values in a field. For example, when set for all fields, the expression (`* * * * *`) means the workflow runs every minute of every hour, on every day, in every month, and on every day of the week.
```

Comment:
Does Asterisk mean wildcard in this context, can you share some reference to this?

Reply:
https://en.wikipedia.org/wiki/Cron#:~:text=Asterisk%20(%20*%20),second%20when%20seconds%20are%20supported.

Reply:
Another - https://docs.oracle.com/cd/E19455-01/805-7229/sysrescron-62861/index.html